Readme.md

```diff
@@ -0,0 +1,181 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ABIY-PORTFOLIO] <a name="about-project"></a>
+
+**[Abiy-Portfolio]** This is a capstone created during my Microverse course. 
+The capstone is able to function (responsive to) both on the mobile and desktop version.
+
+- [Project Description Link](https://www.loom.com/share/4d72f614a9334efc9e8d2958394eafa6)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/Javascript">Javascript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Technologies</summary>
+  <ul>
+    <li><a href="https://dart.dev/tools/linter-rules">LINTERS</a></li>
+  </ul>
+</details>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abiy Hussen**
+
+- GitHub: [@githubhandle](https://github.com/abiy006)
+- Twitter: [@twitterhandle](https://twitter.com/Abiy62463489)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/abiy-hussen-aman-194587183/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[linter_implementation_with_html_css_javaScript]**
+- **[linter_implementation_with_gitignore]**
+- **[implementing_standard_readme_file]**
+- **[implementing_figma_design_background]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://abiy006.github.io/Capstone-1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+* To clone the repository
+* Double click the index.html file and open in your favourite browser
+
+### Prerequisites
+
+In order to run this project you need:
+
+* To clone the repository
+* A browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/abiy006/Capstone-One.git
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[linter_implementation_with_javascript]**
+- [ ] **[linter_implementation_with_html_css_javascript]**
+- [ ] **[linter_implementation_with_ruby]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Walkthrough
+
+Brife description of this portofolio walkthrough until at this point 
+
+- Walkthrough: [Project Walkthrough](https://www.loom.com/share/fb12380aa2964c4f9641a18f9aea6c19)
+
```

Comment:
- Hey, you've added this content that is not relevant to your current project, please, kindly remove it to make your readme more professional.